### PR TITLE
Fix nits

### PR DESCRIPTION
**Pull request by grittygrease**

### CODE REVIEW

File: draft-ietf-tls-exported-authenticator.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ requirements of a Certificate message in the negotiated version of TLS.  In
 particular, the certificate chain MUST be valid for the a signature algorithms
 indicated by the peer in the "signature_algorithms" and "signature_algorithms_cert"
 extension, as described in Section 4.2.3 of {{!TLS13}} for TLS 1.3 or the "signature_algorithms" extension
-from Sections 7.4.2 and 7.4.6 of {{!RFC5246}} for TLS 1.2.
+from Sections 7.4.2 and 7.4.6 of {{?RFC5246}} for TLS 1.2.
```
Comment (MikeBishop) [inline draft-ietf-tls-exported-authenticator.md:259]: Can you have a MUST-level requirement described in an informative reference?

**Review by MikeBishop — COMMENTED** [review]
Both of these seem like similar changes, and I have the same question on both.  They change references to be informative, but in the first case the "informative" reference is in the description of a MUST, and in the second it's in the description of how a required piece of the extension is calculated.

Informative references are inherently documents you don't have to read to properly build the protocol, just useful for understanding it.

The second is probably okay, since HMAC is a well-known term.  The first might not be.

File: draft-ietf-tls-exported-authenticator.md [hunk 2]
```diff
@@ -327,7 +327,7 @@ using a value derived from the connection secrets before taking a user-visible a
 
 ### Finished
 
-A HMAC {{!HMAC=RFC2104}} over the hashed authenticator transcript, which is the
+An HMAC {{?HMAC=RFC2104}} over the hashed authenticator transcript, which is the
```
Comment (chris-wood) [inline draft-ietf-tls-exported-authenticator.md:330]: Shouldn't this be a normative reference?

File: draft-ietf-tls-exported-authenticator.md (hunk 1 above)
Comment (grittygrease) [reply draft-ietf-tls-exported-authenticator.md:259]: Good question. The complaint idnits gave was that RFC 5246 was obsoleted by RFC 8446.

**Review by chris-wood — CHANGES_REQUESTED** [review]
RFC8446 references HMAC normatively, so I tend to think we ought to do the same thing here.

**Comment by chris-wood** [comment]
Yeah, I concur with @MikeBishop. I'd leave these alone for now. We can spin a new version later down the road if folks *really* want to change them.

File: draft-ietf-tls-exported-authenticator.md (hunk 2 above)
Comment (grittygrease) [reply draft-ietf-tls-exported-authenticator.md:330]: This was marked as an error by idnits since RFC 2104 is informational.

Comment (MikeBishop) [reply draft-ietf-tls-exported-authenticator.md:330]: It's also in the downref registry.  It'll need to be disclosed in the shepherd write-up, but I don't think it'll raise any eyebrows.

File: draft-ietf-tls-exported-authenticator.md (hunk 1 above)
Comment (MikeBishop) [reply draft-ietf-tls-exported-authenticator.md:259]: Yeah, but this extension still applies for TLS 1.2, so referencing 1.2 is mostly unavoidable.  Nits don't mean that you have to fix them, just that you need a good reason for diverging.  You've got one.

**Pull request closed without merging**